ic-agent/src/agent/builder.rs

```diff
@@ -87,4 +87,11 @@ impl AgentBuilder {
         self.config.ingress_expiry = ingress_expiry;
         self
     }
+
+    /// Allows disabling query signature verification. Query signatures improve resilience but require
```

Comment:
It looks like the default (`verify_query_signatures: false,`) is to disable query signature verification.  But this comment implies the opposite ("Allows disabling..").  


```suggestion
    /// Allows configuration of query signature verification, which is disabled by default. Query signatures improve resilience but require
```

Reply:
good catch - it's actually supposed to be enabled by default.